Testing/ORKTest/ORKTest/TaskFactory.h

```diff
@@ -0,0 +1,93 @@
+//
+//  TaskFactory.h
+//  ORKTest
+//
+//  Created by Ricardo Sanchez-Saez on 1/9/17.
+//  Copyright © 2017 ResearchKit. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#define DefineStringKey(x) static NSString *const x = @#x
+
+DefineStringKey(ConsentTaskIdentifier);
```

Comment:
Sorry, do you mean you think is better to do it as part of this one or of a new one?

Also, if we want to hold on merging this PR before the other PRs get merged, that's no problem either.